src/components/PasswordSetterBox.js

```diff
@@ -54,11 +54,13 @@ class PasswordSetterBox extends Nimiq.Observable {
             <div class="password-strength strength-good   nq-text-s" data-i18n="passwordbox-password-strength-good"  >Ok, that is an average password</div>
```

Comment:
I cannot add a comment to the line above this (too far away from the diff), but the text `That password is too weak` does no longer hold true, because the password is still allowed. Should be changed to `That password is weak` or similar.